bibliography.bib

```diff
@@ -6092,3 +6092,14 @@ @Inbook{Weinzierl2021
 isbn="978-3-030-76194-3",
 doi="10.1007/978-3-030-76194-3_1",
 }
+
+@Misc{Lamprecht2024-giradar,
+  author        = {Anna-Lena Lamprecht},
+  title         = {Thema im Fokus: Forschungssoftware und Research Software Engineering (RSE)},
```

Comment:
```suggestion
  title         = {{T}hema im {F}okus: {F}orschungssoftware und {R}esearch {S}oftware {E}ngineering ({RSE})},
```
There are BibTeX styles that convert title/subtitle/booktitle/titleaddon fields to lowercase or titlecase depending on which bibliographic convention the journal editor follows, regardless of localization information. Curly braces prevent German nouns and acronyms from changing case. We cannot put the entire title inside curly braces, because that would prevent the change of case for the entire field.